marklogic-data-hub-central/ui/src/config/data.config.js

```diff
@@ -108,40 +108,40 @@ const entityTypeProperties = [
   { key: 9, name: 'gender', type: 'string' }
 ];
 
-const testJSONResponse = { 
-  properties: { 
-    propName: { output: '123EAC', sourcedFrom: 'proteinId' },
+const testJSONResponse = {
+  properties: {
+    propName: { output: 'user1@marklogic.com,user2@marklogic.com, ... (300 more)', sourcedFrom: 'proteinId' },
     propAttribute: { output: 'home', sourcedFrom: 'proteinType' },
   },
   targetEntityType: 'Person'
 };
 
-const testJSONResponseWithFunctions = { 
-  properties: { 
+const testJSONResponseWithFunctions = {
+  properties: {
     propName: { output: '123EAC', sourcedFrom: 'proteinId' },
     propAttribute: { output: 'home-NEW', sourcedFrom: "concat(proteinType,'NEW')" },
   },
   targetEntityType: 'Person'
 };
 
 const errorJSONResponse = {
-  properties: { 
+  properties: {
     propId: { errorMessage: 'Invalid lexical value: "123EACtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest"', sourcedFrom: 'proteinId' },
     propAttribute: { output: 'home', sourcedFrom: 'proteinType' },
   },
   targetEntityType: 'Person'
 };
 
-const mapFunctions = { 
-  "echo": { "category": "custom", "signature": "echo(input)" }, 
-  "memoryLookup": { "category": "builtin", "signature": "memoryLookup(input,inputDictionary)" }, 
-  "documentLookup": { "category": "builtin", "signature": "documentLookup(input,inputDictionaryPath)" }, 
-  "parseDate": { "category": "builtin", "signature": "parseDate(value,pattern)" }, 
-  "parseDateTime": { "category": "builtin", "signature": "parseDateTime(value,pattern)" }, 
-  "add-function": { "category": "custom", "signature": "add-function(num1,num2)" }, 
-  "unparsed-text": { "category": "xpath", "signature": "unparsed-text(xs:string)" }, 
-  "month-from-dateTime": { "category": "xpath", "signature": "month-from-dateTime(xs:dateTime?)" }, "seconds-from-dateTime": { "category": "xpath", "signature": "seconds-from-dateTime(xs:dateTime?)" }, 
-  "concat": { "category": "xpath", "signature": "concat(xs:anyAtomicType?)" } 
+const mapFunctions = {
```

Comment:
This file is only used for testing, right? I recommend adding the word "test" to it somewhere - my first though on seeing this was - why isn't the server providing this data to the UI? But I believe it's just mock data for testing purposes. The filename should imply that.